Build failure: gerbv

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

1. Build `gerbv`.
2. It fails.

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/317994380

### Relevant log output

```console
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/k9s4kp7klnlam9czdp76afj6hr3blavw-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: autoreconfPhase
@nix { "action": "setPhase", "phase": "autoreconfPhase" }
autoreconf: export WARNINGS=
autoreconf: Entering directory '.'
autoreconf: running: autopoint --force
Copying file ABOUT-NLS
Copying file config.rpath
Copying file m4/codeset.m4
Copying file m4/extern-inline.m4
Copying file m4/fcntl-o.m4
Copying file m4/gettext.m4
Copying file m4/glibc2.m4
Copying file m4/glibc21.m4
Copying file m4/iconv.m4
Copying file m4/intdiv0.m4
Copying file m4/intl.m4
Copying file m4/intldir.m4
Copying file m4/intlmacosx.m4
Copying file m4/intmax.m4
Copying file m4/inttypes-pri.m4
Copying file m4/inttypes_h.m4
Copying file m4/lcmessage.m4
Copying file m4/lib-ld.m4
Copying file m4/lib-link.m4
Copying file m4/lib-prefix.m4
Copying file m4/lock.m4
Copying file m4/longlong.m4
Copying file m4/nls.m4
Copying file m4/po.m4
Copying file m4/printf-posix.m4
Copying file m4/progtest.m4
Copying file m4/size_max.m4
Copying file m4/stdint_h.m4
Copying file m4/threadlib.m4
Copying file m4/uintmax_t.m4
Copying file m4/visibility.m4
Copying file m4/wchar_t.m4
Copying file m4/wint_t.m4
Copying file m4/xsize.m4
Copying file po/Makefile.in.in
Copying file po/Makevars.template
Copying file po/Rules-quot
Copying file po/boldquot.sed
Copying file po/en@boldquot.header
Copying file po/en@quot.header
Copying file po/insert-header.sin
Copying file po/quot.sed
Copying file po/remove-potcdate.sin
autoreconf: running: aclocal --force 
configure.ac:41: warning: macro 'AM_GNU_GETTEXT' not found in library
configure.ac:42: warning: macro 'AM_GNU_GETTEXT_VERSION' not found in library
autoreconf: configure.ac: tracing
autoreconf: running: libtoolize --copy --force
libtoolize: putting auxiliary files in '.'.
libtoolize: copying file './ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIRS, 'm4'.
libtoolize: copying file 'm4/libtool.m4'
libtoolize: copying file 'm4/ltoptions.m4'
libtoolize: copying file 'm4/ltsugar.m4'
libtoolize: copying file 'm4/ltversion.m4'
libtoolize: copying file 'm4/lt~obsolete.m4'
libtoolize: Consider adding '-I m4' to ACLOCAL_AMFLAGS in Makefile.am.
autoreconf: configure.ac: not using Intltool
autoreconf: configure.ac: not using Gtkdoc
autoreconf: running: aclocal --force 
configure.ac:41: warning: macro 'AM_GNU_GETTEXT' not found in library
configure.ac:42: warning: macro 'AM_GNU_GETTEXT_VERSION' not found in library
autoreconf: running: /nix/store/i6vhgvkyljmgncgwff2jl1f0qyw2gjjb-autoconf-2.72/bin/autoconf --force
autoreconf: running: /nix/store/i6vhgvkyljmgncgwff2jl1f0qyw2gjjb-autoconf-2.72/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
configure.ac:29: installing './compile'
configure.ac:41: installing './config.guess'
configure.ac:41: installing './config.sub'
configure.ac:28: installing './install-sh'
configure.ac:28: installing './missing'
Makefile.am: installing './INSTALL'
src/Makefile.am: installing './depcomp'
parallel-tests: installing './test-driver'
autoreconf: Leaving directory '.'
autoreconfPhase completed in 48 seconds
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Updating Autotools / GNU config script to a newer upstream version: ./config.sub
Updating Autotools / GNU config script to a newer upstream version: ./config.guess
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
fixing libtool script ./ltmain.sh
./configure
patching script interpreter paths in ./configure
./configure: interpreter directive changed from "#! /bin/sh" to "/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/sh"
configure flags: --disable-static --disable-dependency-tracking --prefix=/nix/store/8mfyknmir1iyyjnw28z1zv1sba4l4nq0-gerbv-2.10.0 --disable-update-desktop-database
checking for a BSD-compatible install... /nix/store/d75200gb22v7p0703h5jrkgg8bqydk5q-coreutils-9.8/bin/install -c
checking whether sleep supports fractional seconds... yes
checking filesystem timestamp resolution... 0.01
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /nix/store/d75200gb22v7p0703h5jrkgg8bqydk5q-coreutils-9.8/bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking xargs -n works... yes
checking whether UID '30010' is supported by ustar format... yes
checking whether GID '30000' is supported by ustar format... yes
checking how to create a ustar tar archive... gnutar
checking whether make supports the include directive... yes (GNU style)
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C... yes
checking whether gcc accepts -g... yes
checking for gcc option to enable C11 features... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... none
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for wchar.h... yes
checking for minix/config.h... no
checking whether it is safe to define __EXTENSIONS__... yes
checking whether _XOPEN_SOURCE should be defined... no
checking whether to enable maintainer-specific portions of Makefiles... no
checking whether NLS is requested... yes
checking for a sed that does not truncate output... /nix/store/k06ssckzrzn9jjvvs4n62m6567zmbx6x-gnused-4.9/bin/sed
checking for msgfmt... /nix/store/1dax567dy7cf55x25a1ryg6z6hl3lvzw-gettext-0.25.1/bin/msgfmt
checking for gmsgfmt... /nix/store/1dax567dy7cf55x25a1ryg6z6hl3lvzw-gettext-0.25.1/bin/msgfmt
checking for xgettext... /nix/store/1dax567dy7cf55x25a1ryg6z6hl3lvzw-gettext-0.25.1/bin/xgettext
checking for msgmerge... /nix/store/1dax567dy7cf55x25a1ryg6z6hl3lvzw-gettext-0.25.1/bin/msgmerge
checking build system type... x86_64-pc-linux-gnu
checking host system type... x86_64-pc-linux-gnu
checking for ld used by gcc... ld
checking if the linker (ld) is GNU ld... yes
checking for shared library run path origin... done
checking how to run the C preprocessor... gcc -E
checking for egrep -e... /nix/store/qfmq7p42ak5yn389qvx7zpxkan5i4xiy-gnugrep-3.12/bin/grep -E
checking for CFPreferencesCopyAppValue... no
checking for CFLocaleCopyCurrent... no
checking for GNU gettext in libc... yes
checking whether to use NLS... yes
checking where the gettext function comes from... libc
checking for po4a-gettextize... no
checking for po4a-translate... no
configure: WARNING: po4a was not found. If you want to change and compile internationalized documentation, please install po4a
checking for onsgmls... no
checking for nsgmls... no
configure: WARNING: no onsgmls nor nsgmls was found. If you want to change and compile internationalized documentation, please install opensp or sp
checking for windows... no
checking whether build environment is sane... yes
checking for gcc... (cached) gcc
checking whether the compiler supports GNU C... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to enable C11 features... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking dependency style of gcc... (cached) none
checking whether the compiler supports GNU C++... yes
checking whether g++ accepts -g... yes
checking for g++ option to enable C++11 features... none needed
checking dependency style of g++... none
checking whether make sets $(MAKE)... (cached) yes
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /nix/store/k06ssckzrzn9jjvvs4n62m6567zmbx6x-gnused-4.9/bin/sed
checking for grep that handles long lines and -e... /nix/store/qfmq7p42ak5yn389qvx7zpxkan5i4xiy-gnugrep-3.12/bin/grep
checking for egrep... /nix/store/qfmq7p42ak5yn389qvx7zpxkan5i4xiy-gnugrep-3.12/bin/grep -E
checking for fgrep... /nix/store/qfmq7p42ak5yn389qvx7zpxkan5i4xiy-gnugrep-3.12/bin/grep -F
checking for ld used by gcc... ld
checking if the linker (ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... nm
checking the name lister (nm) interface... BSD nm
checking whether ln -s works... yes
checking the maximum length of command line arguments... 4718592
checking how to convert x86_64-pc-linux-gnu file names to x86_64-pc-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-pc-linux-gnu file names to toolchain format... func_convert_file_noop
checking for ld option to reload object files... -r
checking for file... file
checking for objdump... objdump
checking how to recognize dependent libraries... (cached) pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ranlib... ranlib
checking for archiver @FILE support... @
checking for strip... strip
checking command to parse nm output from gcc object... ok
checking for sysroot... no
checking for a working dd... /nix/store/d75200gb22v7p0703h5jrkgg8bqydk5q-coreutils-9.8/bin/dd
checking how to truncate binary pipes... /nix/store/d75200gb22v7p0703h5jrkgg8bqydk5q-coreutils-9.8/bin/dd bs=4096 count=1
checking for mt... no
checking if : is a manifest tool... no
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... no
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... no
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... ld -m elf_x86_64
checking if the linker (ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... no
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for animate... notfound
checking for compare... notfound
checking for composite... notfound
checking for convert... notfound
checking for display... notfound
checking for montage... notfound
checking if all ImageMagick tools needed for the testsuite were found... no.  The testsuite will be disabled because the following
tools from the ImageMagick suite were not found:
 animate compare composite convert display montage
No loss in gerbv functionality should be experienced, you just will not
be able to run the regression testsuite.

checking pkg-config is at least version 0.9.0... yes
checking pkg-config is at least version 0.16... yes
checking for sin in -lm... yes
checking for dlopen in -ldl... yes
checking for cairo >= 1.2.0... yes
checking for gtk+-2.0 >= 2.18.0... yes
checking if dxf should be enabled... no
checking if ElectricFence debugging should be enabled... no
checking for env... /nix/store/d75200gb22v7p0703h5jrkgg8bqydk5q-coreutils-9.8/bin/env
checking for gtk-update-icon-path... true
checking for unistd.h... (cached) yes
checking for getopt.h... yes
checking for string.h... (cached) yes
checking for sys/mman.h... yes
checking for sys/types.h... (cached) yes
checking for sys/stat.h... (cached) yes
checking for stdlib.h... (cached) yes
checking for regex.h... yes
checking for libgen.h... yes
checking for time.h... yes
checking for getopt_long... yes
checking for strlwr... no
checking for realpath... yes
checking for canonicalize_file_name... yes
checking whether canonicalize_file_name must be declared... no
checking whether to build with code coverage support... no
checking for the bindir to datadir relative path... ../share
checking for the bindir to pkgdatadir relative path... ../share/gerbv
checking for the bindir to exec_prefix relative path... ..
checking that generated files are newer than configure... done
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating src/Makefile
config.status: creating src/libgerbv.pc
config.status: creating scheme/Makefile
config.status: creating share/glib-2.0/schemas/Makefile
config.status: creating desktop/Makefile
config.status: creating doc/PNG-print/Makefile
config.status: creating doc/eagle/Makefile
config.status: creating doc/html/Makefile
config.status: creating doc/example-code/Makefile
config.status: creating doc/Makefile
config.status: creating man/Makefile
config.status: creating po/Makefile.in
config.status: creating example/eaglecad1/Makefile
config.status: creating example/nollezappare/Makefile
config.status: creating example/numpres/Makefile
config.status: creating example/jj/Makefile
config.status: creating example/dan/Makefile
config.status: creating example/ekf2/Makefile
config.status: creating example/exposure/Makefile
config.status: creating example/am-test/Makefile
config.status: creating example/cslk/Makefile
config.status: creating example/orcad/Makefile
config.status: creating example/Mentor-BoardStation/Makefile
config.status: creating example/pick-and-place/Makefile
config.status: creating example/polarity/Makefile
config.status: creating example/thermal/Makefile
config.status: creating example/trailing/Makefile
config.status: creating example/Makefile
config.status: creating test/Makefile
config.status: creating test/golden/Makefile
config.status: creating test/inputs/Makefile
config.status: creating win32/Makefile
config.status: creating config.h
config.status: executing depfiles commands
config.status: executing po-directories commands
config.status: creating po/POTFILES
config.status: creating po/Makefile
config.status: executing libtool commands

** Configuration summary for gerbv 2.10.0:

   PREFIX:                   
   xdg data directory:       ${datarootdir}

   CPPFLAGS:                  $(subst -I/usr/include/gtk-2.0,-isystem /usr/include/gtk-2.0,$(GTK_CFLAGS))  -DPREFIXDIR=\"${prefix}\" -DBINDIR=\"${bindir}\"
   CFLAGS:                   -g -O2 -Wall  $(subst -I/usr/include/gtk-2.0,-isystem /usr/include/gtk-2.0,$(GTK_CFLAGS)) -I/nix/store/wd6lqkzpll686i5q9snd1c9c7glhcqqi-cairo-1.18.4-dev/include/cairo -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include/freetype2 -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include
   LDFLAGS:                  
   LIBS:                     -ldl -lm   -L/nix/store/733dh2bc55l8khcnkanmz1f99m77xjm0-cairo-1.18.4/lib -L/nix/store/va64srrrxljdrkh4w7y97gl3rzj6dgj9-fontconfig-2.17.1-lib/lib -L/nix/store/cvckjvajk2pdlx7i573fnd7ycrl46f17-freetype-2.13.3/lib -L/nix/store/f7rcazhd826xlcz43il4vafv28888cgj-glib-2.86.3/lib -L/nix/store/5gdqpjsscm98xx8vkwhqahlpgclz87kg-gtk+-2.24.33/lib -L/nix/store/jgyrwzvdvkkwpsx8ca9y0q3ijpn3r4q4-at-spi2-core-2.58.2/lib -L/nix/store/5k9nfx38ckb0wkwsgy925ddzdhrvj16s-gdk-pixbuf-2.44.4/lib -L/nix/store/7qlq78jh6n35v1xpxyd6c279nbf0r88f-pango-1.57.0/lib -L/nix/store/40b7x2n2yylv79rgnavw3xk7yp9308jx-harfbuzz-12.2.0/lib -lgtk-x11-2.0 -lgdk-x11-2.0 -lpangocairo-1.0 -latk-1.0 -lcairo -lgdk_pixbuf-2.0 -lgio-2.0 -lpangoft2-1.0 -lpango-1.0 -lgobject-2.0 -lglib-2.0 -lharfbuzz -lfontconfig -lfreetype -L/nix/store/733dh2bc55l8khcnkanmz1f99m77xjm0-cairo-1.18.4/lib -lcairo -lm

   GTK Version:              2.24.33
   Cairo Version:            1.18.4

   DXF via dxflib:           no

   Electric Fence Debugging: no

   ImageMagick:              no
   (for test suite)

configurePhase completed in 51 seconds
Running phase: buildPhase
@nix { "action": "setPhase", "phase": "buildPhase" }
build flags: SHELL=/nix/store/lw117lsr8d585xs63kx5k233impyrq7q-bash-5.3p3/bin/bash
make  all-recursive
make[1]: Entering directory '/build/source'
Making all in src
make[2]: Entering directory '/build/source/src'
if test -f authors.c.tmp ; then rm -f authors.c.tmp ; fi
awk -F: '/^[ \t]*#/ {next}; /^[ \t]*$/ {next}; {print $1}' ../AUTHORS | sed -e 's;%;%%;g' -e 's;\\;\\\\;g' -e 's;";\\";g' -e 's;^;N_(";g' -e 's;$;"),;g' -e 's;N_("");"";g' > authors.c.tmp
echo '/* Generated from AUTHORS file */' > authors.c
echo '/*! \file authors.c' >> authors.c
echo '    \brief Generated at build time authors list.' >> authors.c
echo '    \ingroup gerbv' >> authors.c
echo ' */' >> authors.c
num=`wc -l authors.c.tmp | awk '{print $1 + 1}'` && \
	echo "const gchar * authors_string_array[$num] = {" >> authors.c
cat authors.c.tmp >> authors.c
rm -f authors.c.tmp
echo 'NULL};' >> authors.c
echo '/* Generated from BUGS file */' > bugs.c
echo '/*! \file bugs.c' >> bugs.c
echo '    \brief Generated at build time bugs list.' >> bugs.c
echo '    \ingroup gerbv' >> bugs.c
echo ' */' >> bugs.c
num=`wc -l ../BUGS | awk '{print $1 + 1}'` && \
	echo "const gchar * bugs_string_array[$num] = {" >> bugs.c
sed -e 's;%;%%;g' -e 's;\\;\\\\;g' -e 's;";\\";g' -e 's;^;N_(";g' -e 's;$;"),;g' -e 's;N_("");"";g' ../BUGS >> bugs.c
echo 'NULL};' >> bugs.c
make  all-am
make[3]: Entering directory '/build/source/src'
gcc -DLOCALEDIR=\"/nix/store/8mfyknmir1iyyjnw28z1zv1sba4l4nq0-gerbv-2.10.0/share/locale\" -DHAVE_CONFIG_H -I. -I..   -I/nix/store/wd6lqkzpll686i5q9snd1c9c7glhcqqi-cairo-1.18.4-dev/include/cairo -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include/freetype2 -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include -I/nix/store/pp2fbp19cabbdb2dhslf3qnxbsyyvnxm-fontconfig-2.17.1-dev/include -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include/freetype2 -I/nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include -I/nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/f7rcazhd826xlcz43il4vafv28888cgj-glib-2.86.3/lib/glib-2.0/include -I/nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0 -I/nix/store/5gdqpjsscm98xx8vkwhqahlpgclz87kg-gtk+-2.24.33/lib/gtk-2.0/include -I/nix/store/q1vzxc56fj1pqdipzh89m4mz278n9fiz-at-spi2-core-2.58.2-dev/include/atk-1.0 -I/nix/store/zsa9rgmanx1l9h7i76gfvv440n90vylg-gdk-pixbuf-2.44.4-dev/include/gdk-pixbuf-2.0 -I/nix/store/nd2qj4saqgbakhdnpl0f543lj9lalyfv-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/ayhcy9p6rshhda432gc1j3nqbhfklpi2-harfbuzz-12.2.0-dev/include/harfbuzz  -DPREFIXDIR=\"/nix/store/8mfyknmir1iyyjnw28z1zv1sba4l4nq0-gerbv-2.10.0\" -DBINDIR=\"/nix/store/8mfyknmir1iyyjnw28z1zv1sba4l4nq0-gerbv-2.10.0/bin\" -DBACKEND_DIR='"/nix/store/8mfyknmir1iyyjnw28z1zv1sba4l4nq0-gerbv-2.10.0/share/gerbv/scheme/"' -DSCMSUBDIR='"scheme"'  -g -O2 -Wall  -I/nix/store/wd6lqkzpll686i5q9snd1c9c7glhcqqi-cairo-1.18.4-dev/include/cairo -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include/freetype2 -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include -I/nix/store/pp2fbp19cabbdb2dhslf3qnxbsyyvnxm-fontconfig-2.17.1-dev/include -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include/freetype2 -I/nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include -I/nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/f7rcazhd826xlcz43il4vafv28888cgj-glib-2.86.3/lib/glib-2.0/include -I/nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0 -I/nix/store/5gdqpjsscm98xx8vkwhqahlpgclz87kg-gtk+-2.24.33/lib/gtk-2.0/include -I/nix/store/q1vzxc56fj1pqdipzh89m4mz278n9fiz-at-spi2-core-2.58.2-dev/include/atk-1.0 -I/nix/store/zsa9rgmanx1l9h7i76gfvv440n90vylg-gdk-pixbuf-2.44.4-dev/include/gdk-pixbuf-2.0 -I/nix/store/nd2qj4saqgbakhdnpl0f543lj9lalyfv-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/ayhcy9p6rshhda432gc1j3nqbhfklpi2-harfbuzz-12.2.0-dev/include/harfbuzz -I/nix/store/wd6lqkzpll686i5q9snd1c9c7glhcqqi-cairo-1.18.4-dev/include/cairo -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include/freetype2 -I/nix/store/ss8rcralsb3b192371ma7kn9wvl3kdx4-freetype-2.13.3-dev/include -c -o attribute.o attribute.c
In file included from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkobject.h:37,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkwidget.h:36,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkcontainer.h:35,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkbin.h:35,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkwindow.h:36,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkdialog.h:35,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkaboutdialog.h:32,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtk.h:33,
                 from attribute.c:47:
/nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtktypeutils.h:236:1: warning: 'GTypeDebugFlags' is deprecated []8;;https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations]8;;]
  236 | void            gtk_type_init   (GTypeDebugFlags    debug_flags);
      | ^~~~
In file included from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/gobject/gobject.h:26,
                 from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/gobject/gbinding.h:31,
                 from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/glib-object.h:24,
                 from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/gio/gioenums.h:30,
                 from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/gio/giotypes.h:30,
                 from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/gio/gio.h:28,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gdk/gdkapplaunchcontext.h:30,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gdk/gdk.h:32,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtk.h:32:
/nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/gobject/gtype.h:725:1: note: declared here
  725 | {
      | ^
In file included from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtktoolitem.h:31,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtktoolbutton.h:30,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtkmenutoolbutton.h:30,
                 from /nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtk.h:126:
/nix/store/fwh1a336x3v4clydwfa8m4bzlpyc7qrg-gtk+-2.24.33-dev/include/gtk-2.0/gtk/gtktooltips.h:73:3: warning: 'GTimeVal' is deprecated: Use 'GDateTime' instead []8;;https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/Warning-Options.html#index-Wdeprecated-declarations-Wdeprecated-declarations]8;;]
   73 |   GTimeVal last_popdown;
      |   ^~~~~~~~
In file included from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/glib/galloca.h:34,
                 from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/glib.h:32,
                 from /nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/gobject/gbinding.h:30:
/nix/store/j359ndglk11v5j47ra66lc1vxlk2dbjw-glib-2.86.3-dev/include/glib-2.0/glib/gtypes.h:580:8: note: declared here
  580 | struct _GTimeVal
      |        ^~~~~~~~~
attribute.c: In function 'attribute_interface_dialog':
attribute.c:361:37: error: passing argument 10 of 'ghid_spin_button' from incompatible pointer type []8;;https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/Warning-Options.html#index-Wincompatible-pointer-types-Wincompatible-pointer-types]8;;]
  361 |                                     intspinner_changed_cb,
      |                                     ^~~~~~~~~~~~~~~~~~~~~
      |                                     |
      |                                     void (*)(GtkWidget *, void *) {aka void (*)(struct _GtkWidget *, void *)}
attribute.c:196:26: note: expected 'void (*)(void)' but argument is of type 'void (*)(GtkWidget *, void *)' {aka 'void (*)(struct _GtkWidget *, void *)'}
  196 |                   void (*cb_func) (), gpointer data, gboolean right_align,
      |                   ~~~~~~~^~~~~~~~~~~
attribute.c:103:1: note: 'intspinner_changed_cb' declared here
  103 | intspinner_changed_cb (GtkWidget * spin_button, gpointer data)
      | ^~~~~~~~~~~~~~~~~~~~~
attribute.c:383:37: error: passing argument 10 of 'ghid_spin_button' from incompatible pointer type []8;;https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/Warning-Options.html#index-Wincompatible-pointer-types-Wincompatible-pointer-types]8;;]
  383 |                                     dblspinner_changed_cb,
      |                                     ^~~~~~~~~~~~~~~~~~~~~
      |                                     |
      |                                     void (*)(GtkWidget *, void *) {aka void (*)(struct _GtkWidget *, void *)}
attribute.c:196:26: note: expected 'void (*)(void)' but argument is of type 'void (*)(GtkWidget *, void *)' {aka 'void (*)(struct _GtkWidget *, void *)'}
  196 |                   void (*cb_func) (), gpointer data, gboolean right_align,
      |                   ~~~~~~~^~~~~~~~~~~
attribute.c:113:1: note: 'dblspinner_changed_cb' declared here
  113 | dblspinner_changed_cb (GtkWidget * spin_button, gpointer data)
      | ^~~~~~~~~~~~~~~~~~~~~
attribute.c:415:71: error: passing argument 8 of 'ghid_check_button_connected' from incompatible pointer type []8;;https://gcc.gnu.org/onlinedocs/gcc-15.2.0/gcc/Warning-Options.html#index-Wincompatible-pointer-types-Wincompatible-pointer-types]8;;]
  415 |                                                TRUE, FALSE, FALSE, 0, set_flag_cb,
      |                                                                       ^~~~~~~~~~~
      |                                                                       |
      |                                                                       void (*)(GtkToggleButton *, gboolean *) {aka void (*)(struct _GtkToggleButton *, int *)}
attribute.c:251:37: note: expected 'void (*)(void)' but argument is of type 'void (*)(GtkToggleButton *, gboolean *)' {aka 'void (*)(struct _GtkToggleButton *, int *)'}
  251 |                              void (*cb_func) (),
      |                              ~~~~~~~^~~~~~~~~~~
attribute.c:81:1: note: 'set_flag_cb' declared here
   81 | set_flag_cb (GtkToggleButton * button, gboolean * flag)
      | ^~~~~~~~~~~
make[3]: *** [Makefile:700: attribute.o] Error 1
make[3]: Leaving directory '/build/source/src'
make[2]: *** [Makefile:532: all] Error 2
make[2]: Leaving directory '/build/source/src'
make[1]: *** [Makefile:489: all-recursive] Error 1
make[1]: Leaving directory '/build/source'
make: *** [Makefile:421: all] Error 2
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.12.56-hardened1, NixOS, 26.05 (Yarara), 26.05.20251228.c0b0e0f`
 - multi-user?: `no`
 - sandbox: `yes`
 - version: `nix-env (Determinate Nix 3.15.1) 2.33.0`
 - nixpkgs: `/nix/store/a47in4d39ig0vxdv8p3lsxbllpl1vvny-source`

### Notify maintainers

@mogorman

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/applications/science/electronics/gerbv/0001-fix-invalid-function-signatures.patch
+++ b/pkgs/applications/science/electronics/gerbv/0001-fix-invalid-function-signatures.patch
@@ -1,0 +1,93 @@
+From ef744d7b8a663f5b9a3e40233eccf8243773e864 Mon Sep 17 00:00:00 2001
+From: Samuel Rounce <me@samuelrounce.co.uk>
+Date: Fri, 23 Jan 2026 01:55:55 +0000
+Subject: [PATCH] fix invalid function signatures
+
+---
+ src/attribute.c |  4 ++--
+ src/callbacks.c | 12 ++++++------
+ 2 files changed, 8 insertions(+), 8 deletions(-)
+
+diff --git a/src/attribute.c b/src/attribute.c
+index 67a622d..6d207bb 100644
+--- a/src/attribute.c
++++ b/src/attribute.c
+@@ -193,7 +193,7 @@ static void
+ ghid_spin_button (GtkWidget * box, GtkWidget ** spin_button, gfloat value,
+ 		  gfloat low, gfloat high, gfloat step0, gfloat step1,
+ 		  gint digits, gint width,
+-		  void (*cb_func) (), gpointer data, gboolean right_align,
++		  void (*cb_func) (GtkWidget *, void *), gpointer data, gboolean right_align,
+ 		  gchar * string)
+ {
+   GtkWidget *hbox = NULL, *label, *spin_but;
+@@ -248,7 +248,7 @@ ghid_check_button_connected (GtkWidget * box,
+ 			     gboolean expand,
+ 			     gboolean fill,
+ 			     gint pad,
+-			     void (*cb_func) (),
++			     void (*cb_func) (GtkToggleButton *, gboolean *),
+ 			     gpointer data, gchar * string)
+ {
+   GtkWidget *b;
+diff --git a/src/callbacks.c b/src/callbacks.c
+index 6b4c90e..9428bde 100644
+--- a/src/callbacks.c
++++ b/src/callbacks.c
+@@ -1066,7 +1066,7 @@ callbacks_toggle_layer_visibility_activate (GtkMenuItem *menuitem, gpointer user
+ 	if (screenRenderInfo.renderType <= GERBV_RENDER_TYPE_GDK_XOR) {
+ 		render_refresh_rendered_image_on_screen ();
+ 	} else {
+-		render_recreate_composite_surface (screen.drawing_area);
++		render_recreate_composite_surface ();
+ 		callbacks_force_expose_event_for_screen ();
+ 	}
+ }
+@@ -2185,7 +2185,7 @@ callbacks_layer_tree_visibility_toggled (gint index)
+ 	if (screenRenderInfo.renderType <= GERBV_RENDER_TYPE_GDK_XOR) {
+ 		render_refresh_rendered_image_on_screen ();
+ 	} else {
+-		render_recreate_composite_surface (screen.drawing_area);
++		render_recreate_composite_surface ();
+ 		callbacks_force_expose_event_for_screen ();
+ 	}
+ }
+@@ -2437,7 +2437,7 @@ callbacks_remove_layer_button_clicked (GtkButton *button, gpointer user_data)
+ 		if (screenRenderInfo.renderType <= GERBV_RENDER_TYPE_GDK_XOR) {
+ 			render_refresh_rendered_image_on_screen ();
+ 		} else {
+-			render_recreate_composite_surface (screen.drawing_area);
++			render_recreate_composite_surface ();
+ 			callbacks_force_expose_event_for_screen ();
+ 		}
+ 	}
+@@ -2468,7 +2468,7 @@ callbacks_move_layer_down_button_clicked  (GtkButton *button, gpointer   user_da
+ 			render_refresh_rendered_image_on_screen ();
+ 		}
+ 		else {
+-			render_recreate_composite_surface (screen.drawing_area);
++			render_recreate_composite_surface ();
+ 			callbacks_force_expose_event_for_screen ();
+ 		}
+ 	}
+@@ -2497,7 +2497,7 @@ callbacks_move_layer_up_button_clicked  (GtkButton *button, gpointer   user_data
+ 			render_refresh_rendered_image_on_screen();
+ 		}
+ 		else {
+-			render_recreate_composite_surface (screen.drawing_area);
++			render_recreate_composite_surface ();
+ 			callbacks_force_expose_event_for_screen ();
+ 		}
+ 	}
+@@ -2525,7 +2525,7 @@ void callbacks_layer_tree_row_inserted (GtkTreeModel *tree_model, GtkTreePath  *
+ 				render_refresh_rendered_image_on_screen();
+ 			}
+ 			else {
+-				render_recreate_composite_surface (screen.drawing_area);
++				render_recreate_composite_surface ();
+ 				callbacks_force_expose_event_for_screen ();
+ 			}
+ 			/* select the new line */
+-- 
+2.52.0
+

--- a/pkgs/applications/science/electronics/gerbv/default.nix
+++ b/pkgs/applications/science/electronics/gerbv/default.nix
@@ -23,6 +23,10 @@ stdenv.mkDerivation rec {
     hash = "sha256-sr48RGLYcMKuyH9p+5BhnR6QpKBvNOqqtRryw3+pbBk=";
   };
 
+  patches = [
+    ./0001-fix-invalid-function-signatures.patch
+  ];
+
   postPatch = ''
     sed -i '/AC_INIT/s/m4_esyscmd.*/${version}])/' configure.ac
   '';

